server/routes/api/subscriptions/subscriptions.ts

```diff
@@ -146,7 +143,13 @@ router.get(
 
     authorize(user, "delete", subscription);
 
-    await subscription.destroy({ transaction });
+    ctx.context = {
```

Comment:
For email unsubscribe flow, this enables the background tasks. Should this be removed?

Reply:
I don't love having the auth type as `API`, will make it indistinguishable from a true API request. Can `auth.type` be left empty?

Otherwise we can also use `createContext` helper here

Reply:
I wasn't sure of using `APP` vs `API` (email action being an one-off case for now) - will use `createContext` here.